### PR TITLE
Update firewall-requirements.md

### PR DESCRIPTION
**Pull request by guanghuthegreat**
adding the requirement for adding AKS Arc's firewall rules.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@alkohli 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/concepts/firewall-requirements.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ This article describes how to optionally use a highly locked-down firewall confi
 
 ## Required firewall URLs
 
-The following table provides a list of required firewall URLs. Make sure to include these URLs to your allowlist.
+The following table provides a list of required firewall URLs. Make sure to include these URLs to your allowlist. 
+
+Please also follow the required firewall requirement of [AKS on Azure Stack HCI](/azure/aks/hybrid/aks-hci-network-system-requirements#firewall-url-exceptions).
```
Comment (sethmanheim) [inline azure-stack/hci/concepts/firewall-requirements.md:41]:
```suggestion
Please also follow the required firewall requirements for [AKS on Azure Stack HCI](/azure/aks/hybrid/aks-hci-network-system-requirements#firewall-url-exceptions).
```

**Comment by alkohli** [comment]
@guanghuthegreat looks good. Thanks!

File: azure-stack/hci/concepts/firewall-requirements.md [hunk 2]
```diff
@@ -58,7 +60,7 @@ The following table provides a list of recommended firewall URLs. If your outbou
 
 Depending on additional Azure services you enable on HCI, you may need to make additional firewall configuration changes. Refer to the following links for information on firewall requirements for each Azure service:
 
-- [AKS on Azure Stack HCI](/azure-stack/aks-hci/system-requirements?tabs=allow-table#aks-on-azure-stack-hci-requirements)
+- [AKS on Azure Stack HCI](/azure/aks/hybrid/aks-hci-network-system-requirements#firewall-url-exceptions)
```
Comment (pradipd) [inline azure-stack/hci/concepts/firewall-requirements.md:63]: Is one of these links the ARB?

**Comment by pradipd** [comment]
Should we also fill up the list in the excel spread sheet? 
https://learn.microsoft.com/en-us/azure-stack/hci/concepts/firewall-requirements#required-firewall-urls

File: azure-stack/hci/concepts/firewall-requirements.md (hunk 2 above)
Comment (guanghuthegreat) [reply azure-stack/hci/concepts/firewall-requirements.md:63]: It added a new link to https://learn.microsoft.com/en-us/azure/aks/hybrid/aks-hci-network-system-requirements#firewall-url-exceptions , and ask customer to follow those addresses making sure AKS Arc works.  The list has been validated by engineering team should include links to ARB.

**Comment by mgodfre3** [comment]
> adding the requirement for adding AKS Arc's firewall rules.
I just want to make sure we are not only adding the Update to the Excel sheet, but also the Table in the doc itself, looking at the changes, I dont see this.

**Comment by alkohli** [comment]
#sign-off

**Comment by alkohli** [comment]
@Jak-MS the PR is signed off.

**Pull request merged**